src/pages/ContentfulPage.vue

```diff
@@ -114,7 +116,8 @@ const getComponentFromType = type => {
 		case 'homepageBottomCTA':
 			return HomepageBottomCTA;
 		case 'homepageHero':
```

Comment:
Should we move this key to something more generic?

Reply:
I think so, now or later is fine though...

Reply:
Yeah, something more generic for this would be good. Maybe 'dynamicHero'? And it might be better to make the change sooner while there aren't many existing Content Groups with that type to update.